groovy/postInit/components/Photomasks.groovy

```diff
@@ -162,7 +162,7 @@ crafting.addShaped("rubylith_ic", metaitem('stencil.ic'), [
     [null, null, null]
 ]);
 
-UV_LIGHT_BOX.recipeBuilder()
+LASER_ENGRAVER.recipeBuilder()
```

Comment:
keep the mask recipe using UV lightbox, since Laser Engraver is gated by Tungsten, an EV material.